File: Motocitizen/src/motocitizen/draw/accidentList/CommonRow.java
package motocitizen.draw.accidentList;

import android.content.Context;
import android.view.ViewGroup;

import motocitizen.accident.Accident;
import motocitizen.main.R;

public class CommonRow extends Row {
    public CommonRow(Context context, Accident accident, ViewGroup parent) {
        super(context, R.layout.accident_row, accident, parent);
    }

    @Override
    public void makeHidden() {
        makeHidden(R.drawable.accident_row_hidden);
    }

    @Override
    public void makeEnded() {
        makeEnded(R.drawable.accident_row_ended);
    }
}

File: Motocitizen/src/motocitizen/rows/accident/CommonRow.kt
package motocitizen.rows.accident

import android.content.Context
import motocitizen.content.accident.Accident
import motocitizen.main.R

abstract class CommonRow(context: Context, accident: Accident) : Row(context, accident) {
    override val margins = arrayOf(4, 2, 16, 2)
    override val layout: Int = R.layout.accident_row
}
